hurricane_project/models.py

```diff
@@ -5,3 +5,7 @@
 # class Login_feature(models.Model):
 #     username =
 #     password =
+
+
+
+# testing pull request (9/16)
```

Comment:
Nice use of comment. Make sure you have a blank line at the end of every Python file.

Reply:
@rlconley fixed